src/main/java/seedu/address/ui/PersonViewCard.java

```diff
@@ -0,0 +1,85 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import seedu.address.model.person.Person;
+import seedu.address.model.schedule.Schedule;
+
+/**
+ * An UI component that displays information of a {@code Person} in the viewed version.
+ */
+public class PersonViewCard extends UiPart<Region> {
```

Comment:
Would it be easier for people to understand if we call it `ScheduleCard` instead?